src/inference/src/dev/threading/cpu_map_scheduling.cpp

```diff
@@ -57,4 +58,33 @@ std::vector<std::vector<int>> apply_hyper_threading(bool input_value,
     return result_table;
 }
 
+bool update_cpu_pinning(const bool input_pinning,
+                        const bool input_changed,
+                        const std::vector<std::vector<int>>& stream_info_table,
+                        const std::vector<std::vector<int>>& original_proc_type_table) {
+#if (defined(_WIN32) || defined(_WIN64))
+
+    return false;
+
+#endif
+
+#ifdef __linux__
```

Comment:
what's about other platforms? macOS? android?

Reply:
This function is part of CPU map logic which is only enabled on Linux and Windows now.

When we enable CPU map for other platform, we will update this function as well.